### PR TITLE
Prometheus metrics for Atlas challenges

### PR DESCRIPTION
**Pull request by danihodovic**
# Checklist:

- [X] Code follows the style guidelines
- ~[X] Documentation updated~
- [x] Corner cases evaluated and handled
- [X] Automatic tests added/update
- [x] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

### CODE REVIEW

File: src/workers/atlas_worker.js [hunk 1]
```diff
@@ -64,23 +82,29 @@ export default class AtlasWorker extends PeriodicWorker {
   async tryWithChallenge(challenge) {
     try {
       if (this.failedChallengesCache.didChallengeFailRecently(challenge.challengeId)) {
+        this.atlasChallengeMetrics.inc({status: atlasChallengeStatus.failedRecently});
```
Comment (spherefoundry) [inline src/workers/atlas_worker.js]: I'm not sure if we should track this as this is only a result of the `failed` metric and this will generate a lot of noise.

Comment (danihodovic) [reply src/workers/atlas_worker.js]: I'd rather collect as many metrics as possible initially to see what the node is exactly doing.

**Comment by danihodovic** [comment]
Thanks for the review!
